### ISSUE TITLE
training log and loss

### ISSUE THREAD

**Issue by haofuly** [wudongming97/Prompt4Driving#7]
Hi @wudongming97 , may I ask for the training log? Otherwise, I have two questions: (1) in the first few training steps, the loss is 130 and the grad_norm is 3277, is it too big that there is some errors? (2) training time for 8 A100 GPU with batchsize of 1.
Thanks very much!

**Comment by wudongming97** [comment]
1) the big grad_norm number is normal at the start of model training; 
2) It takes about 12 hours.

Sorry, I'm busy with the ICCV deadline these days. And I will update the training log in the near future.

**Comment by haofuly** [comment]
Thanks, I have made it !
